lib/rules/no-global-jquery.js

```diff
@@ -48,8 +53,14 @@ module.exports = {
         }
       },
 
+      VariableDeclarator(node) {
+        jQueryLocalName = isVariableAssigmentJquery(node);
```

Comment:
Doesn't this clobber `jQueryLocalName` for every variable declaration?

Specifically, wouldn't the following have `jQueryLocalName` be `null` in the following example?

```js
let a = jQuery;
let b = 'lolwut';

a('.find-me').click();
```

I may be wrong about ^ though. Either way, could you add a test that shows this properly identifying the `a` usage?

Reply:
Yup, this fails in master. Added passing test in new commit